scripts/convert.sh

```diff
@@ -73,6 +75,11 @@ do
         if [ "$char" = $'r' ] || [ "$char" = $'n' ]; then
             printf "\\\\$char" >> $OUTPUT
         fi
+    elif [ "$char" = $'\n' ] && [ $UGLIFY -eq 0 ]; then
+        # Handle new lines if uglify is not installed
+        printf "\"" >> $OUTPUT
+        printf "\n" >> $OUTPUT
+        printf "\"" >> $OUTPUT
```

Comment:
This is for lines that are just new lines. Its adding a " for the end of the previous line, then new line, then adding a " for the next line below. But I needed to do more testing obviously because some scripts don't work. They compile and run, but don't seem to work.